### ISSUE TITLE
Some emojicons aren't displaying properly

### ISSUE THREAD

**Issue by dimensionmedia** [imath/bp-reactions#7]
When I search for some icons, like "Taco" I get the error you see in my screenshot below (error usually appears when a font isn't loaded or property isn't in the font). However when you submit the activity the Taco appears normal.

Pizza works fine in both the activity "whats-new" and the stream.

I tried this also with a default WordPress theme and problem still occurred. Let me know your thoughts! :-)
![screenshot 2016-06-11 13 35 23](https://cloud.githubusercontent.com/assets/721134/15986596/94ae1412-2fd9-11e6-8ff1-b55552be86f2.png)
![screenshot 2016-06-11 13 35 11](https://cloud.githubusercontent.com/assets/721134/15986597/94ae2c22-2fd9-11e6-9c78-15b2bbd69a7e.png)
![screenshot 2016-06-11 13 34 54](https://cloud.githubusercontent.com/assets/721134/15986598/94b173c8-2fd9-11e6-92ec-5ba3cfbdb0b4.png)

**Comment by imath** [comment]
Taco's fine for me :)

![capture d ecran 2016-06-12 a 01 50 33](https://cloud.githubusercontent.com/assets/1834524/15988003/3b4697a2-3040-11e6-9c2f-3a9723ffb322.png)

What is the browser you're using ? Is this a Microsoft Windows machine ?

**Comment by dimensionmedia** [comment]
Nope. This is a Mac, Firefox (latest) version. You can see in my screenshot that something is off... hmmm... I'll try Chrome. I know the Taco comes out fine on my iPhone (mobile Safari).

**Comment by imath** [comment]
Weird!!

This is what i get in FF

![capture d ecran 2016-06-12 a 01 55 44](https://cloud.githubusercontent.com/assets/1834524/15988019/dcdabe2c-3040-11e6-9ecc-27a3b5ad8e44.png)
